src/spanningTree/spanningTree_process.cpp

```
@@ -0,0 +1,144 @@
+            } else if (suffix == "DFS") {
+                val = 2;
+            } else if (suffix == "BFS") {
+                val = 3;
+            }
+
+            which = static_cast<Which>(static_cast<int>(which) + val);
+
+            switch (val) {
+                case 1:
+                    if (distance < 0) {
+                        pgr_throw_error("Negative value found on 'distance'", "Must be positive");
```

Cppcheck (2.19.0)

[style] The function 'pgr_free' is never used.

(unusedFunction)